plugins.json

```diff
@@ -1,26 +1,26 @@
 [
-    {
-        "name": "Node Manager",
-        "description": "Install and manage Massa nodes from Thyra",
-        "assets": {
-            "windows": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_windows-amd64.zip",
-                "checksum": "02ee5479b18149300815af3ec1b0f6ac"
-            },
-            "linux": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_linux-amd64.zip",
-                "checksum": "4b7ba25d612ea41066a8cc679420a60d"
-            },
-            "macos-arm64": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_darwin-arm64.zip",
-                "checksum": "3feb252f7a57e8a393d5ccfc1604f2ea"
-            },
-            "macos-amd64": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_darwin-amd64.zip",
-                "checksum": "247655ea2985c488c678e195699684d7"
-            }
-        },
-        "version": "0.0.8",
-        "url": "https://github.com/massalabs/thyra-node-manager-plugin"
+  {
+    "name": "Node Manager",
+    "description": "Install and manage Massa nodes from Thyra",
+    "assets": {
+      "windows": {
+        "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_windows-amd64.zip",
```

Comment:
Just ensure that we wont end up with massa-station paths...

Reply:
the `plugins.json` will be modified if the plugin is updated. I didn't make changes here but I'll take your comment into consideration for #6